### PR TITLE
Add placeholder gears as logo

### PR DESCRIPTION
**Pull request by lilyminium**
Adds the logo to MDAKits

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG.md updated?
 - [ ] AUTHORS.md updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by lilyminium** [comment]
Screenshot 2024-06-22 at 11 12 17 pm

Added some docs explaining to edit the logo, and also updated MDAKits placeholder to this.

**Comment by lilyminium** [comment]
@orbeckst is this what you were thinking? I could also drop the question mark in the placeholder if that's simpler!

**Review by orbeckst — APPROVED** [review]
Looking great. I’d leave the template with question mark in, just as you have it. 

Once you’re done with the logo here, can you also add the official MDAkit logo to the branding repo?

(I’m not 100% sure if we should put the empty template into branding as well — it might become confusing. Perhaps have a separate repo for free templates???)

File: tests/test_options.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def test_official_mda_theme(self, tmpdir):
 
             text = conf.read_text()
             assert '"mda_official": True,' in text
-            assert "html_logo" not in text
-            assert "html_favicon" not in text
+            # assert "html_logo" not in text
```
Comment (IAlibay) [inline tests/test_options.py:42]: maybe worth removing these if they are no longer useful?

**Review by IAlibay — APPROVED** [review]
Approving, although with the one comment re: removing commented out test lines.

File: tests/test_options.py (hunk 1 above)
Comment (lilyminium) [reply tests/test_options.py]: Thank you -- done!

**Pull request merged**